src/main/scala/com/mozilla/telemetry/sinks/CrashesBatchHttpSink.scala

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package com.mozilla.telemetry.sinks
+
+import scalaj.http.HttpRequest
+
+import scala.collection.mutable.ArrayBuffer
+
+case class CrashesBatchHttpSink(
```

Comment:
Used to be BatchHttpSink. It's renamed here and refactored to extend HttpSink (used to be standalone).